setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+
+setup(
+    name="ppo",
+    version="0.1",
+    description="Simple PPO implementation on OpenAI GYM environment.",
+    packages=["src"],
```

Comment:
Since we changed  `src` to `ppo`, shouldn't this be changed as well?

Reply:
Yes, definitely. I am going to do that.